packages/build-tools/src/android/__tests__/expoUpdates.test.ts

```diff
@@ -47,7 +47,7 @@ describe(androidSetClassicReleaseChannelNativelyAsync, () => {
   test('sets the release channel', async () => {
     const reactNativeProjectDirectory = fs.mkdtempSync('/expo-project-');
     fs.ensureDirSync(reactNativeProjectDirectory);
-    const releaseChannel = 'default';
+    const releaseChannel = 'blah';
```

Comment:
Can we also include a test for this specific case? So with a releaseChannel that would previously be incorrectly 'cast'?

Reply:
This test checks that the meta data is stored as a "string resource".

A string resource is guaranteed to be interpreted as a string.

Reply:
we can't really test it because the resulting interpreted value is available in android runtime

Reply:
Ah, I misread the way the result is validated. Thanks for the explanation, that makes perfect sense then.